app/styles/_core.less

```diff
@@ -1209,3 +1209,14 @@ copy-to-clipboard .input-group.limit-width {
     width: 100%;
   }
 }
+
+// handle top-border in table rows inside of nested loops
+tbody.nested-loop {
+  > tr {
+    + tr {
+      &:nth-child(2) {
+        border-top-width: 0;
```

Comment:
@rhamilto I was not sure if this was the correct approach to fixing the double border on tables that need a hidden `<tr>` before the actual first visible `<tr>` element. wdyt?